src/IssueViz.Security/Hotspots/LocalHotspotsStore.cs

```diff
@@ -186,8 +185,7 @@ private void RematchAll()
 
         private void InitializeServerHotspots(IList<SonarQubeHotspot> sonarQubeHotspots)
         {
-            serverHotspots = sonarQubeHotspots?.ToList() ?? new List<SonarQubeHotspot>();
-            unmatchedHotspots = serverHotspots.ToHashSet();
```

Comment:
The issue said to remove the field and define it in the methods that used it.
It was only used in one method, and it didn't actually need to be defined as a separate variable.